README.md

```diff
@@ -115,3 +115,78 @@ if (handleStartupEvent()) {
   return;
 }
 ```
+
+## Checking for updates, downloading and installing
```

Comment:
> yes, it exists for osx, but as for windows, I'm not sure.

The auto updater does exist on Windows.

https://github.com/electron/electron/blob/master/lib/browser/api/auto-updater/auto-updater-win.js
https://github.com/electron/electron/blob/master/lib/browser/api/auto-updater/squirrel-update-win.js